<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Using Machine Learning to Predict the Flu- Intro</title>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="style.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
  <script src="main.js"></script>
</head>
<body>
  <div class="header" id="welcome">
    <div class="navbar">
      <ul>
        <li><a href="index.html">Intro</a></li>
        <li>&frasl;</li>
        <li><a href="project.html">Project</a></li>
        <li>&frasl;</li>
        <li><a href="about.html">About</a></li>
      </ul>
    </div>
    <div class="content">
      <h1>Using Machine Learning to Predict the Flu</h1>
      <a href="#background"><h2><i class="fa fa-arrow-circle-down"></i></h2></a>
      <h2>Created by Chris Tan</h2>
    </div>
  </div>
  <div class="body">
    <div class="navbar fixed">
      <span class="align-left">Predicting the Flu with ML</span>
      <a class="toggle-collapse" href="#collapse"><span class="fa fa-bars"></span></a>
      <div id="collapse">
        <ul>
          <li class="active"><a href="#welcome">Welcome</a></li>
          <li><a href="#abstract">Abstract</a></li>
          <li><a href="#questions">Questions</a></li>
          <li><a href="#hypotheses">Hypotheses</a></li>
          <li><a href="#background">Background</a></li>
        </ul>
      </div>
    </div>
    <div class="content">
      <div class="normal">
        <a id="abstract"></a>
        <h1>Abstract</h1>
      </div>
      <div class="alternate">
        <a id="questions"></a>
        <h1>Questions</h1>
        <p>Can we use machine learning to accurately predict virulent strains of the influenza virus?</p>
        <p>How accurate is each machine learning algorithm, and which is best at making predictions?</p>
      </div>
      <div class="normal">
        <a id="hypotheses"></a>
        <h1>Hypotheses</h1>
      </div>
      <div class="alternate">
        <a id="background"></a>
        <h1>Background</h1>
        <p>Every year, about 3 million people are diagnosed with the flu, a viral infection of the respiratory passages. Typically, flu season occurs between October and May, often peaking between December and February. However, receiving annual vaccinations before the seasonal epidemic can help prevent illness.</p>
        <p>The flu is caused by influenza viruses, which infect the mucous cells of the respiratory system. Influenza viruses are part of the virus family called orthomyxoviridae, which are RNA viruses that contain eight segments that code for ten genes. The RNA is bound to proteins to form a helix, which is encapsulated within a viral envelope, embedded in which are two surface glycoproteins, hemagglutinin (HA) and neuraminidase (NA), which are coded for by Segment 4 and Segment 6, respectively.</p>
        <p>To infect mucous cells, hemagglutinin and neuraminidases exploit the N-acetylneuraminic (sialic) acid found in the cell membranes of the host cells. Hemagglutinin is responsible for binding to the host cell, which it recognizes by its sialic acid, while neuraminidase helps prevent the virus particles from agglutinating, or clumping together. Hemagglutinin and neuraminidase are also the antigens of the influenza viruses, by which the immune system can recognize and respond to them. Antibodies, part of the system’s immune response, render virus particles ineffectual by binding to the two surface proteins, preventing them from infecting host cells. The effectiveness of a vaccine is determined by how well the antibodies are able to bind to the antigens.</p>
        <p>Although antibodies typically remain effective for several years, the human population is still susceptible to annual infections due to the rapid evolution of the antigenic regions of the influenza virus, due in no small part to its high mutability. As the antigenic regions grow increasingly different from the form antibodies recognize, the immune response of our bodies becomes less effective. There are two primary forms of these changes in antigenic characteristics: antigenic drift, in which the antigens gradually mutate to become less recognizable and retain some relation to previous strains, and antigenic shift, which results in entirely novel antigens.</p>
        <p>Since there is so much variability among influenza viruses, there is a standardized protocol for naming them to allow for proper identification, which provides important information while analyzing strains. The nomenclature of influenza viruses, a convention accepted by the World Health Organization in 1979, includes: (1) its antigenic type; (2) the host of origin; (3) geographical origin; (4) strain number; (5) year of isolation; (6) for influenza A viruses, the hemagglutinin and neuraminidase antigen description in parentheses. In this experiment, the primary subjects will be influenza A viruses, specifically the H1N1 and H3N2 subtypes, since they are the most prevalent and most virulent.</p>
        <p>To predict the amino acid sequences of H1N1 and H3N2 influenza strains, I will be using machine learning, with relies on human-created algorithms to analyze large quantities of data that the computer will “learn” from. I will be creating two simple algorithms, one additive and the other multiplicative, that will attempt to account for the conserved domains of the hemagglutinin and neuraminidase surface glycoproteins while allowing variability in the antigenic regions, which are subject to change due to the mutability of influenza viruses, to produce amino acid sequences for potentially dangerous H1N1 and H3N2 strains. I will focus on the HA1 subunit of hemagglutinin, where most changes occur for that surface protein, while analyzing the entire neuraminidase protein, since there are multiple different areas in the amino acid sequence where changes may occur.</p>
        <p>To evaluate the accuracy of my algorithms, since I cannot create virulent influenza strains and test them on the human population, I will instead use Basic Local Alignment Search Tool (BLAST) to compare its percent similarity to common strains from previous years, since it is believed that mutations to the influenza genome, and by extension, the rate of antigenic drift, occur at a fairly constant rate, and calculate the percent error, assuming the “actual” rate to be the mean percentage of similarity among common strains of the past. Through this experiment, I hope to illustrate the potential applications of bioinformatics.</p>
      </div>
      <span id="toProject">To see the rest of my project, <a href="project.html">proceed to 'Project'</a></span>
    </div>
  </div>
</body>
</html>
