<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Using Machine Learning to Predict the Flu- Project</title>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="style.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
  <script src="main.js"></script>
</head>
<body>
  <div class="header" id="welcome">
    <div class="navbar">
      <ul>
        <li><a href="index.html">Intro</a></li>
        <li>&frasl;</li>
        <li><a href="project.html">Project</a></li>
        <li>&frasl;</li>
        <li><a href="about.html">About</a></li>
      </ul>
    </div>
    <div class="content">
    </div>
  </div>
  <div class="body">
    <div class="navbar fixed">
      <span class="align-left">Predicting the Flu with ML</span>
      <a class="toggle-collapse" href="#collapse"><span class="fa fa-bars"></span></a>
      <div id="collapse">
        <ul>
          <li><a href="#methods">Methods</a></li>
          <li><a href="#data">Data</a></li>
          <li><a href="#discussion">Discussion</a></li>
          <li><a href="#conclusion">Conclusion</a></li>
          <li><a href="#bibliography">Bibliography</a></li>
        </ul>
      </div>
    </div>
    <div class="content">
      <div class="normal">
        <a id="methods"></a>
        <h1>Methods</h1>
        <p>This requires a computer with the following:
          <ul>
            <li>Internet access</li>
            <li>Command line interface (CLI)</li>
            <li>Python 3.5.<i>x</i></li>
            <li>Python interactive development environment (IDE)</li>
          </ul>
        </p>
      </div>
      <div class="alternate">
        <a id="data"></a>
        <h1>Data</h1>
        <p>To see the full data, click here: <a href="https://tinyurl.com/n6g6q8a">Machine Learning Flu Data</a></p>
      </div>
      <div class="normal">
        <a id="discussion"></a>
        <h1>Discussion</h1>
        <p>The capability of my algorithms to accurately predict potential influenza strains is predicated upon the assumptions that antigenic drift occurs as a steady rate and that certain parts of the genome, and by extension, the amino acid sequence, are conserved, which means percent similarities among common strains should be fairly similar. However, the anomaly of the 2009 H1N1 influenza virus, which is an instance of antigenic shift, most likely skewed my data for the glycoprotein amino acid sequences for H1N1, as well as the mean percent similarities and the calculated percent error. This, however, is not entirely true, as the percent error for hemagglutinin for H1N1 is indeed above 5%, while the percent error for neuraminidase is remarkably low, at about 1.1%. Aside from the H1N1 hemagglutinin proteins, the percent error for my algorithms were all below 5%.</p>
        <p>As shown by my data, there was little discrepancy between the additive and multiplicative algorithms, with neither being significantly more accurate than the other. This may be due to both relying heavily on probability, since I was attempting to conserve a significant portion of the genome. However, my algorithms overestimated the rate of antigenic drift, as their percent similarities tended to be lower than what the actual data demonstrated. This was likely due to the use of randomization, which models random mutations, but does not account for the selective pressure in reality.</p>
      </div>
      <div class="alternate">
        <a id="conclusion"></a>
        <h1>Conclusion</h1>
        <p>My algorithms were able to almost nearly match the actual rate of antigenic drift, assuming it is relatively constant, conserving certain parts of the genome while allowing for random mutations that lead to changes in the antigens, preventing the immune systems from recognizing it. Thus, the strains produced by the computer could possibly be virulent, and so the predictions can be deemed accurate.</p>
        <p>Based on BLAST analyses, the strains produced by the computer were less similar to the common strains than the common strains were to each other, suggesting an overestimation of the rate of antigenic drift. However, the percent error was mostly below 5%, with the only exceptions being due to an anomalous flu strain.</p>
        <p>While it is only a rough framework at this point, it could certainly be improved to account for more factors that would lead to more accurate predictions that would be anticipatory as opposed to reactionary, as our vaccinations currently are. For instance, additional biases accounting for time of origin or tendencies of polypeptides could contribute to more accurate algorithms. Essentially, this experiment demonstrates the potential applications of bioinformatics to solve complex problems in medicine and biology. </p>
      </div>
      <div class="normal">
        <a id="bibliography"></a>
        <h1>Bibliography</h1>
        <p>Bouvier, Nicole M., and Peter Palese. “THE BIOLOGY OF INFLUENZA VIRUSES.” Vaccine 26.Suppl 4 (2008): D49–D53. Print.</p>
        <p>Couch, Robert B. "Orthomyxoviruses." Medical Microbiology. 4th Edition. U.S. National Library of Medicine, 01 Jan. 1996. Web.</p>
        <p>Gamblin, Steven J., and John J. Skehel. “Influenza Hemagglutinin and Neuraminidase Membrane Glycoproteins.” The Journal of Biological Chemistry 285.37 (2010): 28403–28409. PMC. Web. 9 Apr. 2017.</p>
        <p>Gelderblom, Hans R. "Structure and Classification of Viruses." Medical Microbiology. 4th Edition. U.S. National Library of Medicine, 01 Jan. 1996. Web.</p>
        <p>Shtyrya, Y.A., L.V. Mochalova, and N.V. Bovin. “Influenza Virus Neuraminidase: Structure and Function.” Acta Naturae 1.2 (2009): 26–32. Print.</p>
        <p>SRIWILAIJAROEN, Nongluk, and Yasuo SUZUKI. “Molecular Basis of the Structure and Function of H1 Hemagglutinin of Influenza Virus.” Ed. Hiroshi KIDA. Proceedings of the Japan Academy. Series B, Physical and Biological Sciences 88.6 (2012): 226–249. PMC. Web. 9 Apr. 2017.</p>
        <p>Zhirnov, Oleg P., Mine R. Ikizler, and Peter F. Wright. “Cleavage of Influenza A Virus Hemagglutinin in Human Respiratory Epithelium Is Cell Associated and Sensitive to Exogenous Antiproteases.” Journal of Virology 76.17 (2002): 8682–8689. PMC. Web. 9 Apr. 2017.</p>
      </div>
    </div>
  </div>
</body>
</html>
